packages/data-broker/src/__tests__/services/superset/SupersetService.test.ts

```diff
@@ -41,149 +41,139 @@ describe('SupersetService', () => {
     it('throws an error', () => expect(supersetService.delete()).toBeRejectedWith('not supported'));
   });
 
-  describe('pull()', () => {
-    describe('pullAnalytics()', () => {
-      it('pulls', () =>
-        expect(
-          supersetService.pull([DATA_ELEMENTS.ITEM_1], 'dataElement', DEFAULT_PULL_OPTIONS),
-        ).resolves.toEqual({
-          metadata: {
-            dataElementCodeToName: {},
+  describe('pullAnalytics()', () => {
+    it('pulls', () =>
```

Comment:
Lifiting test cases one level up & using `.pullAnalytics()` instead of `.pull()`